sdk/search/Azure.Search.Documents/src/Generated/DataSourcesRestClient.cs

```diff
@@ -76,6 +76,7 @@ internal HttpMessage CreateCreateOrUpdateRequest(string dataSourceName, SearchIn
             request.Headers.Add("Prefer", "return=representation");
             request.Headers.Add("Accept", "application/json; odata.metadata=minimal");
             request.Headers.Add("Content-Type", "application/json");
+            request.Headers.Add("Accept", "application/json");
```

Comment:
You duplicated the header. We already attach it, with `odata.metadata=none` or `odata.metadata=minimal` depending on the client.

Reply:
Good catch. https://github.com/Azure/autorest.modelerfour/issues/335